src/main/java/uk/gov/hmcts/darts/audio/component/impl/AudioRequestSummaryMapperImpl.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.darts.audio.component.impl;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.audio.component.AudioRequestSummaryMapper;
+import uk.gov.hmcts.darts.audio.service.impl.AudioRequestSummaryResult;
+import uk.gov.hmcts.darts.audiorequests.model.AudioRequestSummary;
+import uk.gov.hmcts.darts.audiorequests.model.MediaRequestStatus;
+
+import java.util.List;
+
+@Component
+public class AudioRequestSummaryMapperImpl implements AudioRequestSummaryMapper {
+
+    @Override
+    public List<AudioRequestSummary> mapToAudioRequestSummary(
+        List<AudioRequestSummaryResult> results) {
+        return results.stream()
+            .map(this::mapToAudioRequestSummary)
+            .toList();
+    }
+
+    private AudioRequestSummary mapToAudioRequestSummary(AudioRequestSummaryResult result) {
```

Comment:
Mapstruct can probably be used here if you want.

Reply:
I've left this conversion as is - as I'm not familiar with mapstruct and just followed something similar to what Chris B. did recently this sprint